bin/aurelia.js

```diff
@@ -56,17 +56,15 @@ function handleCommands(env) {
   }
 
   aurelia.init({
-    env : env
+      env : env
+    , argv: argv
```

Comment:
@ahmedshuhel I need access to argv before we run aurelia.run()

So I am passing it on to the config.

Yes I know it's on config.env.argv, 
This should be a flat and easily access property.